src/expire.c

```diff
@@ -149,16 +149,16 @@ void expireScanCallback(void *privdata, void *entry) {
     data->sampled++;
 }
 
-static inline int isExpiryTableValidForSamplingCb(hashtable *ht) {
+static inline int expiryTableShouldSkipForSamplingCb(hashtable *ht) {
```

Comment:
Good, but maybe reorder the words for better grammar? "Should skip table for sampling" is better than "Table should skip for sampling" I think.

```suggestion
static inline int expiryShouldSkipTableForSamplingCb(hashtable *ht) {
```

Oh, is "expiry" the prefix for this file, or is "expiryTable" one concept? Maybe it should be one of these?

1. expireShouldSkipTableForSamplingCb (prefix "expire" like the filename)
2. shouldSkipExpiryTableForSamplingCb (if expiryTable is one concept)

Reply:
expireShouldSkipTableForSamplingCb seems better to me.
I don't see we have or use expiryTable this concept a lot